libkode/printer.cpp

```diff
@@ -956,16 +951,47 @@ void Printer::printImplementation( const File &file, bool createHeaderInclude )
     filename.prepend( d->mOutputDirectory + '/' );
 
   QFile implementation( filename );
-  if ( !implementation.open( QIODevice::WriteOnly ) ) {
-    qWarning( "Can't open '%s' for writing.", qPrintable( filename ) );
-    return;
+  printCodeIntoFile(out, &implementation);
+}
+
+void Printer::printCodeIntoFile(const Code &code, QFile *file)
```

Comment:
missing spaces inside (...), for consistency with the rest of the code (as much as I don't like it)

[happens on some other lines as well, like 822, 954, 969, 970, and the very first line modified by this commit]

Reply:
Fixed.